### ISSUE TITLE
emojilib

### ISSUE THREAD

**Issue by Mottie** [remy/textexpander-emoji-snippets#1]
Awesome work as always!

Please consider combining the work in this `emoji.json` with the [emojilib](https://github.com/muan/emojilib)'s [`emojis.json`](https://github.com/muan/emojilib/blob/master/emojis.json) file.

The emojilib file includes an indicator for emojis that work with a fitzpatrick scale modifier whereas this file contains unicode and ios version information.

**Comment by remy** [comment]
I should have really included where I sourced the emoji from!

What is the fitzpatrick scale modifier (and moreover, what is it for WRT this project?).

**Comment by Mottie** [comment]
The [Fitzpatrick scale](https://en.wikipedia.org/wiki/Fitzpatrick_scale) was used to define an extra unicode character which may be added to modify the skin tone of a particular set of emojis - there are [five tones](http://emojipedia.org/modifiers/); a total of six different colors including the emoji without a modifier.

Here is an example of Santa with & without a Fitzpatrick modifier:

| Emoji | Fitzpatrick <br>Tone | Fitzpatrick<br>Name/Type | Unicode |
|:---:|:----:|:-----|---------|
| 🎅 |    | No modifier           | 🎅 U+1F385 | 
| 🎅🏻 | 🏻 | Light tone (type 1-2) | 🎅 U+1F385 🏻 U+1F3FB |
| 🎅🏼 | 🏼 | Medium-Light (type 3) | 🎅 U+1F385 🏼 U+1F3FC |
| 🎅🏽 | 🏽 | Medium tone (type 4)  | 🎅 U+1F385 🏽 U+1F3FD |
| 🎅🏾 | 🏾 | Medium dark (type 5)  | 🎅 U+1F385 🏾 U+1F3FE |
| 🎅🏿 | 🏿 | Dark (type 6)         | 🎅 U+1F385 🏿 U+1F3FF |

> Note: Support of modifiers for each emoji might be different depending on the OS.

From https://github.com/muan/emojilib/pull/111

**Comment by remy** [comment]
Of course. I always just think of this as the tone modifier, forgetting it
had a name. I'll see if I can change the script so it reads this property
and adds modifiers (or probably smarter: making default tone collections).
